Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM python:3.12-slim
 LABEL "maintainer" "Sviatoslav Sydorenko <wk+pypa@sydorenko.org.ua>"
 LABEL "repository" "https://github.com/pypa/gh-action-pypi-publish"
 LABEL "homepage" "https://github.com/pypa/gh-action-pypi-publish"
+LABEL "org.opencontainers.image.source" "https://github.com/pypa/gh-action-pypi-publish"
```

Comment:
I'd accept this in a separate PR even before the rest is figured out/refactored.

Reply:
Moved to separate PR (#241).